test/functional/job_configs.py

```diff
@@ -58,6 +67,9 @@ def __init__(self, config, expected_to_fail, expected_num_subjobs, expected_num_
         - TOKEN: seq 0 4 | xargs -I {} echo "This is atom {}"
 
 """,
+        'nt': """
+""",
```

Comment:
@josephharrington  I have one last job config to rewrite anyway. I will update the PR with all batch commands under key 'nt'.